text-generation-inference/server/text_generation_server/models/causal_lm.py

```diff
@@ -536,20 +571,73 @@ def __init__(
             padding_side="left",
             truncation_side="left",
         )
-        model = AutoModelForCausalLM.from_pretrained(
-            model_id,
-            revision=revision,
-            torch_dtype=dtype,
-        )
+
+        model_kwargs = {
+            "revision": revision,
+            # "token": args.token,
+        }
+
+        world_size = int(os.getenv("WORLD_SIZE", "1"))
+        rank = int(os.getenv("RANK"), 0)
+        self.enable_hpu_graph = os.getenv("ENABLE_HPU_GRAPH", "true").lower() == "true"
```

Comment:
Don't we want to enable HPU graphs anyway?

Reply:
we set default as hpu graph, and give user an option also